### ISSUE TITLE
USER ISSUE: Black screen when walking

### ISSUE THREAD

**Issue by GSnakeUS** [StrangeLoopGames/EcoIssues#7292]
**Version:** 0.7.3.3 beta

**Steps to Reproduce:**
walking over bridge

**Expected behavior:**
everything normal

**Actual behavior:**
screen went black, interface chat map etc. visible but can't see the world, it's black

**Comment by SeanAtr0n** [comment]
Could you send us your log file?
paste this into explorer to find it `%APPDATA%\..\LocalLow\Strange Loop Games\ECO`

**Comment by Elliander** [comment]
This happened to me once, however it happened to coincide with a chrome extension updating in the background and when a new tab was automatically created the screen went black until I restarted the game.

**Comment by goertzenator** [comment]
Something similar has happened to me:
- Was walking along a long bridge with a cart and the screen went black for a few seconds.  It did come back so I kept going.
- At the end of the bridge there was supposed to be a down ramp, but it was missing and the screen went black again permanently.
- All logins take me to a black screen (attached).
- My coordinates are listed as (-2147483648, -2147483648, -2147483648).
- /unstuck does nothing, as does 'e'.
- Another player reported my character present and turning in the game while I am in this black world. 
 Server is called "The Upside Down".  Very fitting.
- Version 7.4.7.

output_log.txt is
```
Initialize engine version: 2017.4.2f2 (52d9cb89b362)
GfxDevice: creating device client; threaded=1
Direct3D:
    Version:  Direct3D 11.0 [level 11.1]
    Renderer: Radeon (TM) RX 480 Graphics (ID=0x67df)
    Vendor:   ATI
    VRAM:     4041 MB
    Driver:   23.20.15002.11
Begin MonoManager ReloadAssembly
- Completed reload, in  0.077 seconds
<RI> Initializing input.

<RI> Input initialized.

<RI> Initialized touch support.

UnloadTime: 0.862151 ms
Setting up 4 worker threads for Enlighten.
  Thread -> id: 7174 -> priority: 1 
  Thread -> id: 631c -> priority: 1 
  Thread -> id: 5864 -> priority: 1 
  Thread -> id: 658c -> priority: 1 
The referenced script on this Behaviour (Game Object 'Camera') is missing!
 
(Filename:  Line: 1758)

WARNING: Shader Unsupported: 'CalmWater/Calm Water [Double Sided]' - Pass 'FORWARDADD' has no vertex shader
WARNING: Shader Unsupported: 'CalmWater/Calm Water [Double Sided]' - All passes removed
Client version: 0.7.4.7 beta
Thursday, June 21, 2018 4:26 PM
 
(Filename: C:\buildslave\unity\build\artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

Changing state: PreLogin -> LoggedIn
 
(Filename: C:\buildslave\unity\build\artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

Changing state: LoggedIn -> Connecting
 
(Filename: C:\buildslave\unity\build\artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

Changing state: Connecting -> PasswordRequest
 
(Filename: C:\buildslave\unity\build\artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

Changing state: PasswordRequest -> Connecting
 
(Filename: C:\buildslave\unity\build\artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

The referenced script on this Behaviour is missing!
 
(Filename:  Line: 1515)

The referenced script on this Behaviour (Game Object 'WorldObjects') is missing!
 
(Filename:  Line: 1758)

Missing following objects: AirPumpObject, APGenObject, AtmosphereRegulatorObject, AtmosphereTesterObject, BigStorageChestObject, ButtonObject, GasGeneratorObject, LogicGateObject, SwitchObject
 
(Filename: C:\buildslave\unity\build\artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

Changing state: Connecting -> FadeAndClose
 
(Filename: C:\buildslave\unity\build\artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

Screen position out of view frustum (screen pos 0.000000, 0.000000, 300.000000) (Camera rect 0 0 1720 720)
 
(Filename:  Line: 3124)

Screen position out of view frustum (screen pos 1720.000000, 0.000000, 300.000000) (Camera rect 0 0 1720 720)
 
(Filename:  Line: 3124)

Screen position out of view frustum (screen pos 0.000000, 0.000000, 960.000000) (Camera rect 0 0 256 256)
 
(Filename:  Line: 3124)

Screen position out of view frustum (screen pos 256.000000, 0.000000, 960.000000) (Camera rect 0 0 256 256)
 
(Filename:  Line: 3124)

Screen position out of view frustum (screen pos 0.000000, 0.000000, 960.000000) (Camera rect 0 0 256 256)
 
(Filename:  Line: 3124)

Screen position out of view frustum (screen pos 256.000000, 0.000000, 960.000000) (Camera rect 0 0 256 256)

<eliding approx 3.7 million lines of similar logfile>

Screen position out of view frustum (screen pos 0.000000, 0.000000, 300.000000) (Camera rect 0 0 128 128)
 
(Filename:  Line: 3124)

Screen position out of view frustum (screen pos 128.000000, 0.000000, 300.000000) (Camera rect 0 0 128 128)
 
(Filename:  Line: 3124)

Screen position out of view frustum (screen pos 128.000000, 128.000000, 300.000000) (Camera rect 0 0 128 128)
 
(Filename:  Line: 3124)

Screen position out of view frustum (screen pos 0.000000, 128.000000, 300.000000) (Camera rect 0 0 128 128)
 
(Filename:  Line: 3124)

[34:33] Local server manager destroyed, stopping server.Releasing render texture that is set as Camera.targetTexture!
 
(Filename:  Line: 3926)
```

![20180624162356_1](https://user-images.githubusercontent.com/605072/41824027-06311fb4-77cf-11e8-99f4-e9f8156e70a7.jpg)

**Comment by SeanAtr0n** [comment]
should be fixed in 7.6
